integration_tests/cairo_zero_hint_tests/uint256_improvements.small.cairo

```diff
@@ -78,4 +295,11 @@ func main{range_check_ptr}() {
     assert sign = 0;
 
     return ();
-}
\ No newline at end of file
+}
+
```

Comment:
why is `uint128_mul` function never called?

Reply:
copied entire integration test